src/tutorials/lua_basics_lightbulb.md

```diff
@@ -130,50 +211,61 @@ Your rotations may be different, but will most likely look like this:
 
 ### Create Variables for Each Rotation
 
-Now that you know the rotations of the switch for "on" and "off", you can save them as variables. On the next line of your `LightToggleScript` add these lines:
+Now that you know the rotations of the switch for "on" and "off", you can save them as variables. On the next line of your `LightSwitchToggle` add these lines:
 
 ```lua
-local onRotation = Rotation.New(0, -60, -0)
-local offRotation = Rotation.New(-180, -60, 180)
+local ROTATION_ON = Rotation.New(0, -60, 0)
+local ROTATION_OFF = Rotation.New(0, -130, 0)
 ```
 
 - `Rotation.New` means we are telling the script to rotate our object to a new set of coordinates.
-- `(0, 60, 0)` are the x, y, and z coordinates (respectively) of where we want our switch to rotate to. We want to rotate our switch up along the y-axis by 90 degrees.
+- `(x, y, z)` are the coordinates (respectively) of where we want our switch to rotate to. We want to rotate our switch up along the y-axis by 90 degrees.
 
 !!! note
     You will almost always use `Rotation.New` when rotating an object, but there is also a predefined Rotation called `Rotation.ZERO` which will rotate the object to `0, 0, 0`.
     [Learn more about Rotations on the Core API](../api/rotation.md) page.
 
-### Make the Switch Move with a Script
+!!! note
+    It is convention that constants, variables that will not be altered, are in **SCREAMING_SNAKE_CASE**, meaning all caps with **_** between each word.
+
+### Make the Switch Rotate with a Script
 
 Next you will rotate the switch. On a new line, type:
 
 ```lua
-switch:RotateTo(onRotation, 2)
+local TIME_ROTATE = 2
+
+Switch:RotateTo(ROTATION_ON, TIME_ROTATE)
 ```
 
-- `switch` tells the script to rotate the object attached to this variable.
-- `RotateTo` is an function that tells Core we want to rotate an object.
-- `2` is the time it should take to complete the rotation.
+- `Switch` tells the script to rotate the object attached to this variable.
+- `RotateTo(rotation, time)` is an function that tells Core we want to rotate an object.
+- `TIME_ROTATE` is the time it should take to complete the rotation. Storing this as a variable makes it really easy to edit in the future.
 
 The script should now look like this:
 
 ```lua
-local switch = script.parent
+local Switch = script:GetCustomProperty("Switch"):WaitForObject()
+local Light = script:GetCustomProperty("Light"):WaitForObject()
+local Trigger = script:GetCustomProperty("Trigger"):WaitForObject()
 
-local onRotation = Rotation.New(0, -60, -0)
-local offRotation = Rotation.New(-180, -60, 180)
+local ROTATION_ON = Rotation.New(0, -60, 0)
+local ROTATION_OFF = Rotation.New(0, -130, 0)
+local TIME_ROTATE = 2
 
-switch:RotateTo(onRotation, 2)
+Switch:RotateTo(ROTATION_ON, TIME_ROTATE)
 ```
 
+!!! note
+    It is best to avoid **magic numbers** whenever possible. **Magic Numbers** are direct usage of numbers in code, such as if we did `Switch:RotateTo(Rotation.New(0, -60, 0), 0.1)`.
```

Comment:
I kind of like "magic numbers" here, but start with the positive: Using variables makes it easier to change later, because you don't have to find out where in your code you change the rotation to change what you want the rotation to be.

Maybe something in the magic numbers about how you might not remember why you chose that number, and the variable also lets say what it represents, as in TIME_ROTATE